src/harness/reference_models/aggregate_interference/aggregate_interference.py

```diff
@@ -0,0 +1,362 @@
+
+  return aggregate_interference
+
+def aggrFSSBlockingConstraint(protection_point, low_freq, high_freq, 
+                              grant_objects, fss_info, protection_ent_type):
+  """ Calculates aggregate interference from CBSDs in the neighborhood of 
+  FSS Blocking pass band
+  """ 
+
+  # Assign values to the protection constraint
+  constraint = interf.ProtectionConstraint(latitude=protection_point[0],
+                 longitude=protection_point[1], low_frequency=low_freq,
+                 high_frequency=high_freq, entity_type=protection_ent_type)
+
+  # Identify CBSD grants in the neighborhood of the protection constraint
+  neighborhood_grants = interf.findOverlappingGrantsInsideNeighborhood(grant_objects, constraint)
+
+  cbsd_interference_list = []
+  aggregate_interference = {}
+
+  # Get interference caused by each grant
+  for grant in neighborhood_grants:
+    cbsd_interference = interf.computeInterferenceFssBlocking(grant, constraint,
+                          fss_info, grant.max_eirp)
+    cbsd_interference_list.append(cbsd_interference)
+
+  aggregate_interference[constraint] = convertAndSumInterference(cbsd_interference_list)
+  return aggregate_interference
+
+
+def calculateAggregateInterferenceForFss(fss_record, cbsd_list):
+  """Calculates aggregate interference for FSS.
+  
+  Calculates aggregate interference for each protection constraint in the FSS
+  based on the interferences caused by CBSD's grants in the neighborhood of
+  the FSS protection entity.
+
+  Args:
+    fss_record: FSS protection entity
+    cbsd_list: list of CBSD objects containing RegistrationRequests and GrantRequests
+  Returns:
+    Aggregate interference to FSS in the dictionary format,where the key is a tuple. 
+    {(latitude, longitude, low_frequency, high_frequency,entity_type):
+    aggregate_interference(dBm)}
+  """
+  grant_list = interf.getAllGrantInformationFromCbsdDataDump(cbsd_list)
+  # Get FSS T&C Flag value
+  fss_weight1 = fss_record['deploymentParam'][0]['weight1']
+  fss_weight2 = fss_record['deploymentParam'][0]['weight2']
+
+  fss_ttc_flag = fss_record['deploymentParam'][0]['ttc']
+  # Get the protection point of the FSS
+  fss_point = fss_record['deploymentParam'][0]['installationParam']
+  protection_point = (fss_point['latitude'], fss_point['longitude'])
+
+  # Get the frequency range of the FSS
+  fss_freq_range = fss_record['deploymentParam'][0]['operationParam']['operationFrequencyRange']
+  fss_low_freq = fss_freq_range['lowFrequency']
+  fss_high_freq = fss_freq_range['highFrequency']
+
+  # Get FSS information
+  fss_info = interf.FssProtectionPoint(latitude=fss_point['latitude'], 
+               longitude=fss_point['longitude'],
+               height_agl=fss_point['height'],
+               max_gain_dbi=fss_point['antennaGain'],
+               pointing_azimuth=fss_point['antennaAzimuth'],
+               pointing_elevation=fss_point['antennaElevationAngle'],
```

Comment:
we need to decide once for all what defined the pointing_elevation, and be coherent in the json. 
Is it : antennaElevationAngle or - antennaDowntilt (minus here)

Reply:
Use antennaDowntilt